src/main/assembly/dist/bin/easy-sword2-service-start

```diff
@@ -3,6 +3,8 @@
 BINPATH=`command readlink -f $0 2> /dev/null || command grealpath $0 2> /dev/null`
 APPHOME=`dirname \`dirname $BINPATH \``
 
+# Set LC_ALL to a locale with UTF-8 to make sure non-ASCII file names are written correctly to the file system (see: EASY-1254).
+LC_ALL=en_US.UTF-8 \
 java -Dlogback.configurationFile=$APPHOME/cfg/logback-service.xml \
      -Dapp.home=$APPHOME \
-     -cp $APPHOME/bin/easy-bag-store.jar nl.knaw.dans.easy.sword2.Sword2Service
+     -cp $APPHOME/bin/easy-sword2.jar nl.knaw.dans.easy.sword2.Sword2Service
```

Comment:
Wauw! That is an interesting catch! This script apparently never gets executed...